src/Sulu/Component/PHPCR/PathCleanup.php

```diff
@@ -55,44 +50,16 @@ public function __construct(array $replacers)
      */
     public function cleanup($dirty, $languageCode = null)
     {
-        $replacers = $this->replacers['default'];
-
-        if (null !== $languageCode) {
-            $replacers = array_merge(
-                $replacers,
-                (isset($this->replacers[$languageCode]) ? $this->replacers[$languageCode] : [])
-            );
-        }
-
-        if (count($replacers) > 0) {
-            foreach ($replacers as $key => $value) {
-                $dirty = str_replace($key, $value, $dirty);
-            }
+        $parts = explode('/', $dirty);
+        $newParts = [];
+        foreach ($parts as $part) {
+            $part = str_replace('&', '-and-', $part);
```

Comment:
I am sorry, but this is not gonna cut it. We can't just replace it with "and", but the "and" has to be translated.

English: "You & I" => "you-and-i"
German: "Du & Ich" => "du-und-ich"
Spanish: "Tu & yo" => "tu-y-yo"

Do you get the gist? That's something we might still need the `replacers.xml` file for, if the transliterator/urlizer or whoever is not taking care of that.

Reply:
@danrot that's the case only for ResourceLocator, right? In Urlizer I don't see code that handles these.

Reply:
Well, if we implement this I would say we also handle the node names this way. Don't see any reason to introduce two different ways here.

If the Urlizer is not handling this (have you tested that?) we probably have to keep the `replacers.xml` file with just these few replacements defined.

Reply:
Yup, I'm sure. Just tested it. You & I becomes **you-and**. That's current behaviour, with replacers (btw replacers aren't used in Urlizer)

Reply:
Yup, just tested it. **You & I** becomes **you-i**. Btw replaces are not used in Urlizer at all.

Reply:
As written I would because of BC not remove the replacers part as somebody could have used it to replace special things in its url generation.

So I would go with keeping this part:

https://github.com/sulu/sulu/blob/94fb1b2f5f6b7566c5f3765800567ed7bc0f0cfe/src/Sulu/Component/PHPCR/PathCleanup.php#L58-L71

and then call the slugger after that. We could maybe remove the replaces which are not longer necessary but would keep the logic of replacing.